app/assets/stylesheets/header.css.scss

```diff
@@ -119,3 +118,32 @@ li:hover ul {
   top: 0;
   width: 669px;
 }
+
+.banner-information {
+  box-shadow: 10px 10px 30px 0 rgba(0, 0, 0, 0.75);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.75)` should only be used in variable declarations; they should be referred to via variable everywhere else.